include/rocksdb/advanced_options.h

```diff
@@ -644,6 +644,18 @@ struct AdvancedColumnFamilyOptions {
   // Dynamically changeable through SetOptions() API
   uint64_t ttl = 0;
 
+  // Bottommost level files older than this TTL will be picked up for
+  // compaction. The data in the L_max level might be needed around for a much
+  // longer time before requring a cleanup in some use cases. And hence the need
+  // for introducing a separate ttl option for bottommost level.
+  // Supported only in Level compaction.
+  // Pre-req: ttl > 0 && max_open_file == -1.
+  // unit: seconds. Ex: 7 days = 7 * 24 * 60 * 60
+  // Default: 0 (disabled)
+  //
+  // Dynamically changeable through SetOptions() API
+  uint64_t bottommost_level_ttl = 0;
```

Comment:
Is there a better way we represent this option? I feel it is hard for users to reason about ttl for other levels and bottommost level and figure out what it means for their data.
How if we call it something like time_to_compact or something like that, and apply it to every file, not just the file from the last level? In this way, users know that all files will be reompacted after N seconds, and the compaction filter will be applied. I know it's not a great interface either, but worth considering.